core/src/test/java/io/substrait/extendedexpression/ProtoExtendedExpressionConverterTest.java

```diff
@@ -0,0 +1,78 @@
+package io.substrait.extendedexpression;
+
+import io.substrait.TestBase;
+import io.substrait.expression.Expression;
+import io.substrait.expression.ExpressionCreator;
+import io.substrait.expression.FieldReference;
+import io.substrait.expression.ImmutableFieldReference;
+import io.substrait.proto.ExtendedExpression;
+import io.substrait.type.ImmutableNamedStruct;
+import io.substrait.type.Type;
+import io.substrait.type.TypeCreator;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ProtoExtendedExpressionConverterTest extends TestBase {
+  static final String NAMESPACE = "/functions_arithmetic_decimal.yaml";
+
+  @Test
+  public void fromTest() throws IOException {
```

Comment:
It might be better to have a single `roundtripExtendedExpression` test so that you can test both to/from proto and compare the full objects afterwards (instead of `toProtoTest` and `fromProtoTest`).

Reply:
Just maintain only roundtrip class for Expression, Aggregation and Expression + Aggregation